doc/source/howtos/upgrade_guide.rst

```diff
@@ -117,3 +120,17 @@ To upgrade the cluster, follow these steps for each node:
 #. Repeat this procedure for the next node in the cluster
    until you upgrade all nodes.
 
+.. note:: It is important that when rejoining, the node should synchronize using
+   :term:`IST`. If for some reason :term:`SST` syncronization is used
+   instead, there is a problem, as the node will use data-directory from
+   the previous version, and newer version binaries would throw a lot of
+   errors about wrong structure. This :term:`SST`-related problem is
+   garanteed in case of major upgrade, and is possible in case of minor
+   upgrade also.
+
+   If this problem occurs, it can be solved with the following steps: 
+   * shutdown node that rejoined cluster using SST
+   * restart in standalone
+   * run mysql-upgrade
+   * restart in cluster mode. (ensure cluster joins back using IST).
+	 
```

Comment:
Also, we should introduce this section as part of relevant steps. 
Currently, upgrade enlist 9 steps. These steps are post 9 where-in user check and run extra steps (10-13).